sympy/polys/tests/test_ring_series.py

```diff
@@ -108,6 +108,31 @@ def test_inversion():
     raises(ZeroDivisionError, lambda: rs_series_inversion(p, x, 3))
 
 
+def test_series_reversion_newton():
+    x,a = symbols('x,a')
+    # exponential function with parameter
+    f = exp(x*a)-1
+    n = 4
+    f = f.series(x,0,n).removeO()
+    Rx = f.as_poly(x,field=True).domain[x]
+    fr = Rx.from_sympy(f)
+    xr = Rx.from_sympy(x)
+    ar = Rx.domain.from_sympy(a)
+    calculated = rs_series_reversion_newton(fr,xr,n)
+    expected = 1/ar/3*xr**3 - 1/ar/2*xr**2 + 1/ar*xr
+    assert calculated == expected
+    # linear function with parameter
+    f = x*a
+    n = 2
+    f = f.series(x,0,n).removeO()
+    Rx = f.as_poly(x,field=True).domain[x]
+    fr = Rx.from_sympy(f)
+    xr = Rx.from_sympy(x)
+    ar = Rx.domain.from_sympy(a)
+    calculated = rs_series_reversion_newton(fr,xr,n)
+    expected = 1/ar*xr
```

Comment:
This code reads to me as if it was possibly supposed to mean `1/(ar*xr)` which is not the same. Usually this would be like `xr/ar` but of course if you want to emphasise that it is polynomial in xr then perhaps `(1/ar)*xr` is better.

Reply:
Changed in f06c9c071eb6d04b2e7e0aefc76212849d2d98fa